elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -0,0 +1,414 @@
+<link rel="stylesheet" href="https://patternflyelements.com/core/styles/pfe-layouts.min.css">
+<link rel="stylesheet" href="/elements/rh-secondary-nav/demo/demo.css"/>
+<link rel="stylesheet" href="/elements/rh-secondary-nav/rh-secondary-nav-lightdom.css"/>
+
+
+<pfe-band size="small" color="darkest">
+  <h2 slot="header">Note</h2>
+  <p>We dont have container queries yet, so this compnent's media query break points are set in understanding that the component will exist at full screen width.  The breakpoints will appear off if the component isn't 100% page width. You can use the fullscreen button above to test media query breakpoint.  However doing so will break position: sticky of the menu.</p>
+</pfe-band>
+
+<rh-secondary-nav role="navigation">
+  <a href="#" slot="logo" id="logo-id"> <!-- id is upgraded if not given -->
+    Red Hat Ansible Automation Platform <!-- <svg> or text can be added here -->
+  </a>
+  <ul slot="nav" aria-labelledby="logo-id"> <!-- is this ul labelledby the text or above what if its a svg? -->
+    <li>
+      <rh-secondary-nav-dropdown>
```

Comment:
I've got this itch in the back of my neck that says "pssst... abstract this into `/elements`..."

but also YAGNI, we'll worry about that when the time comes

Reply:
Can you explain this a bit more?  Are you suggesting getting rid of the `<rh-secondary-nav-dropdown>` element and abstracting it into the shadow dom so to speak?  If so let's have a conversation on that.  That should be something i should be able to tackle.